README.md

```diff
@@ -0,0 +1,126 @@
+# General registry
+
+The official registry of general Julia packages. This registry of packages is
```

Comment:
This seems awkward "of general Julia packages". I like better the one in the registry description,
> Official general Julia package registry where people can
register any package they want without too much debate about
naming and without enforced standards on documentation or
testing. We nevertheless encourage documentation, testing and
some amount of consideration when choosing package names.

Reply:
I agree it sounds a bit awkward; it is the tagline of the repo and possibly could be changed there as well. I will update the README.